docs/shared_model_manager.md

```diff
@@ -11,13 +11,15 @@ Models are only loaded into memory when needed using the `fetch_model` function.
 
 The usage example demonstrates adding models and then using them with their respective functionalities.
 
+⚠️ ❕: we should ALWAYS **add model instance to CPU** , and model pool will automatically put it in GPU.
```

Comment:
didn't quite understand this phrase, if we set to CPU why it will be automatically put it GPU?

Reply:
Lemme explain more in doc